src/features/bulk/store/selectors.ts

```diff
@@ -0,0 +1,28 @@
+import { createSelector } from '@reduxjs/toolkit';
+import { IBulkMetadataState, IBulkMetadataField, IBulkMetadataFile } from './model';
+
+export const initialState: IBulkMetadataState = {
+	currentStep: 'upload',
+	files: [],
+	fields: [],
+};
+
+export const getState = (state: IBulkMetadataState) => state;
+
+export const getFilesFromStore = createSelector(
```

Comment:
ik zou hier ook selectors verwachten om een metadataveld/waarde combi op te halen voor een bepaald bestand.

Reply:
Ja, hierbij loop ik wel tegen een probleem aan dat de metadata verdeelt staat over een individuele file en in de fields array in de bulk-upload state.